src/Umbraco.Tests.AcceptanceTest/cypress/integration/Members/members.js

```diff
@@ -24,8 +25,8 @@ context('Members', () => {
 
         cy.get('input#_umb_login').clear().type(email);
         cy.get('input#_umb_email').clear().type(email);
-        cy.get('input#password').clear().type(password);
-        cy.get('input#confirmPassword').clear().type(password);
+        cy.get('input#password').clear().type(password, { timeout: passwordTimeout });
```

Comment:
I think it is okay in these tests to just type a shorter password. We wait too long for a simple password :)

Reply:
Ah that's true, it does seem like an unnecessarily long password for no particular reason, I'll just cut it half. 
It might be a good idea to keep  the extended timeout since it doesn't cause the test to take longer, and I have had some weird cases when cypress stops typing for a little bit causing even small strings to take longer than 4 seconds, as mentioned in the part about the helper package in my description.